### PR TITLE
Content update 

### PR DESCRIPTION
**Pull request by timahenning**
Updated the content.  Issue #29

### CODE REVIEW

File: Hands-on lab/Before the HOL - Intelligent analytics.md [hunk 1]
```diff
@@ -47,48 +47,69 @@ Synopsis: In this exercise, you will set up your environment for use in the rest
 
 ### Task: Setup a lab virtual machine 
 
-1. In the [Azure Portal](https://portal.azure.com/), select +Create a resource, then type "Visual Studio" into the search bar. Select Visual Studio Community 2017 on Windows Server 2016 (x64) from the results. 
+1. In the [Azure Portal](https://portal.azure.com/), select +Create a resource, then type "Visual Studio" into the search bar. Select Visual Studio Community 2017 **(latest release)** on Windows Server 2016 (x64) from the results. 
 
-![In the Azure Portal Everything section, under Results, under Name, Visual Studio Community 2017 on Windows Server 2016 is circled.](media/setup5.png 'Azure Portal Everything section')
+![In the Azure Portal Everything section, under Results, under Name, Visual Studio Community 2017 on Windows Server 2016 is circled.](media/2019-03-20-10-21-21.png)
+![](media/2019-03-20-10-27-57.png)
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/Before the HOL - Intelligent analytics.md]: Missing alt-text.

File: Hands-on lab/Before the HOL - Intelligent analytics.md [hunk 2]
```diff
@@ -47,48 +47,69 @@ Synopsis: In this exercise, you will set up your environment for use in the rest
...
 
 2. On the blade that comes up, at the bottom, ensure the deployment model is set to Resource Manager, and select Create.
 
-    ![At the Bottom of the blade, Resource Manager is selected as the deployment model.](media/setup6.png 'Bottom of the blade')
+    ![At the Bottom of the blade, Resource Manager is selected as the deployment model.]
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/Before the HOL - Intelligent analytics.md]: Image not resolving.

File: Hands-on lab/Before the HOL - Intelligent analytics.md [hunk 3]
```diff
@@ -47,48 +47,69 @@ Synopsis: In this exercise, you will set up your environment for use in the rest
...
+7. Click **Next: Networking**
 
-5. On the Choose a size blade, select **DS2_V3 Standard**.
+    ![Review the next blade - Networking.](media/2019-03-20-11-18-33.png)
 
-    ![The Choose a size blade has the D2S_V3 Standard option selected](media/setup-vm-size.png 'Choose a size blade')
+8. Leave defaults
 
-6. Choose **Select** to move on to the Settings blade.
+    ![](media/2019-03-20-11-20-21.png)
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/Before the HOL - Intelligent analytics.md]: Missing alt-text.

**Comment by DawnmarieDesJardins** [comment]
I see a security alert on this one, please advise.

1 bootstrap vulnerability found in …/ChatWebApp/packages.config on Feb 22
Remediation
Upgrade bootstrap to version 3.4.1 or later. For example: 
<package id="bootstrap" version="3.4.1" />
Always verify the validity and compatibility of suggestions with your codebase. 

Details
CVE-2019-8331 
More information
moderate severity 
Vulnerable versions: >= 3.0.0, < 3.4.1 
Patched version: 3.4.1 
In Bootstrap 4 before 4.3.1 and Bootstrap 3 before 3.4.1, XSS is possible in the tooltip or popover data-template attribute. For more information, see: https://blog.getbootstrap.com/2019/02/13/bootstrap-4-3-1-and-3-4-1/

File: Hands-on lab/HOL step-by-step - Intelligent analytics.md [hunk 4]
```diff
@@ -457,30 +483,34 @@ In this section, you will provision an Azure Cosmos DB account, a DocumentDB Dat
...
 5. On the Add Collection blade, enter the following:
 
-    - **Database id**: Enter awhotels.
+    - **Database id**: Create new. Enter awhotels.
 
     - **Collection Id**: Enter messagestore.
 
-    - **Partition Key**: Enter a partition key such as /userid or /australia.
+    - **Partition Key**: Enter a partition key such as **/username**.
+        > Note: Pick a field in this schema.  Otherwise, you will have no documents in the CosmoDB collection. Below is a sample of the messages stored in the CosmoDB at a later part in the lab.
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/HOL step-by-step - Intelligent analytics.md]: Formatting should be >**Note**:

File: Hands-on lab/HOL step-by-step - Intelligent analytics.md [hunk 5]
```diff
@@ -1559,24 +1609,38 @@ In this task, you will configure a Logic App to send notifications to hotel gues
     ![Service Bus - When a message is received in a queue (auto-complete) is highlighted in the Triggers list.](media/logic-app-triggers-service-bus-queue.png "Service Bus Triggers")
 
 5. On the When a message is received in a queue (auto-complete) dialog, select the **awhotel-staff-notifications** queue from the Queue name drop down.
+    > Note: Remember the queue you set up earlier.
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/HOL step-by-step - Intelligent analytics.md]: correct formatting.  "Note" should be bolded

File: Hands-on lab/HOL step-by-step - Intelligent analytics.md [hunk 6]
```diff
@@ -166,6 +166,23 @@ If you are already connected to your Lab VM, skip to Step 6.
...
+        ![Showing IE menu.](media/2019-03-20-15-40-37.png)
+    - Click **Internet Options**
+    - Click the **Security** tab and the **Custom Level** for the **Internet**
+
+    ![Showing the Internet Options](media/2019-03-20-15-44-24.png)
+    - Scroll down to **File download** and enable.
+
+    ![Showing Internet custom options](media/2019-03-20-15-46-49.png)
+
+13. Now you can dowload files from the Internet. Download and install Google Chrome.
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/HOL step-by-step - Intelligent analytics.md]: typo - download

**Comment by timahenning** [comment]
I have made the requested changes.  Let me know if there is anything else.
Tim

**Pull request merged**